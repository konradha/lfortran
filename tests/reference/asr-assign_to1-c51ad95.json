{
    "basename": "asr-assign_to1-c51ad95",
    "cmd": "lfortran --indent --show-asr --implicit-typing --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/assign_to1.f90",
    "infile_hash": "0da8cafe77fcfae28448ae5b873c873d6e643054caf9f348813f834b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-assign_to1-c51ad95.stdout",
    "stdout_hash": "cf3e48f712a6681731c3d68b8e4806d14bd94745c8a0f8b5f5330ba7",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}