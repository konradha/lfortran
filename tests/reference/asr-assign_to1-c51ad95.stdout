(TranslationUnit 
    (SymbolTable
        1
        {
            main:
                (Program 
                    (SymbolTable
                        2
                        {
                            k:
                                (Variable 
                                    2 
                                    k 
                                    Local 
                                    () 
                                    () 
                                    Default 
                                    (Integer 4 []) 
                                    Source 
                                    Public 
                                    Optional 
                                    .false.
                                )
                            
                        }) 
                    main 
                    [] 
                    [(GoToTarget 
                        1000
                    )
                    (If 
                        (IntegerCompare 
                            (IntegerConstant 0 (Integer 4 [])) 
                            Lt 
                            (IntegerConstant 1 (Integer 4 [])) 
                            (Logical 4 []) 
                            (LogicalConstant 
                                .true. 
                                (Logical 4 [])
                            )
                        ) 
                        [(= 
                            (Var 2 k) 
                            (IntegerConstant 1 (Integer 4 [])) 
                            ()
                        )] 
                        []
                    )
                    (GoToTarget 
                        100
                    )
                    (Select 
                        (Var 2 k) 
                        [] 
                        []
                    )
                    (GoToTarget 
                        1
                    )
                    (Print 
                        () 
                        [(StringConstant 
                            "run" 
                            (Character 1 3 () [])
                        )] 
                        () 
                        ()
                    )]
                )
            
        }) 
    []
)
